js/script.js

```diff
@@ -10,7 +10,8 @@ document.addEventListener('DOMContentLoaded', function() {
 });
 
 var prepareTweet = function(input) {
-  if (Array.isArray(input)) { var data = input[0]; }
+  var data = 'No data';
```

Comment:
`'No data'` is a string, so you can't take from string another values. Simplest way to solve this problem is stop the function when input isn't an array